src/index.spec.ts

```diff
@@ -84,15 +84,15 @@ describe('Transactions', () => {
           CardanoWasm.Address.from_bytes(
             Buffer.from('61a6274badf4c9ca583df893a73139625ff4dc73aaa3082e67d6d5d08e', 'hex'),
           ),
-          // we can construct Coin from both BigInt (here) or from a string (below in fee)
-          CardanoWasm.Coin.new(BigInt(1)),
+          // we can construct BigNum (Coin) from both a js BigInt (here) or from a string (below in fee)
```

Comment:
The downside is that when you do `type Coin = BigNum` in rust while you can use it in rust directly like `Coin::new(0)` you can't do that in JS as it only exports the `BigNum` class and replaces all other things that type alias to `BigNum` as `BigNum` such as `Coin` or `TransactionMetadataLabel` and such.

If we want to allow direct usage from js using the aliases we could either copy the code for every u64-like type (lots of duplicate code) or make the entire class into a macro and invoke it a few times (which is kinda ugly imo since there's no real difference in the code) so I'm not sure which approach we want to take between those and the current case of it all being `BigNum` from the js side.